### ISSUE TITLE
Odd Behavior (Channel capping when it shouldn't)

### ISSUE THREAD

**Issue by Velocikitty** [AppliedEnergistics/Applied-Energistics-2#573]
Playing in the recommended branch of DW20's 1.7.10 pack.

![2014-12-10_05 25 34](https://cloud.githubusercontent.com/assets/10140904/5374417/2d6bebd8-802d-11e4-8e73-c39d9ae2ebab.png)
![2014-12-10_05 26 36](https://cloud.githubusercontent.com/assets/10140904/5374419/2d6ce9ac-802d-11e4-8dfa-ed0e22704e51.png)
![2014-12-10_05 26 43](https://cloud.githubusercontent.com/assets/10140904/5374418/2d6c375a-802d-11e4-920a-9552a11e291c.png)

Basically, it is randomly capping and saying its got no more channels, despite being linked into a dense cable into a controller.
